### PR TITLE
RaR: Reboot

### PR DESCRIPTION
**Pull request by jwarwick**
Not positive I'm referencing `state` correctly in the recursive function.

### CODE REVIEW

**Comment by NoahTheDuke** [comment]
Doesn't look wrong, but that doesn't mean it isn't, lol.

File: src/clj/game/cards/events.clj [hunk 1]
```diff
@@ -1544,6 +1544,30 @@
                  (doseq [c (-> @state :runner :temp-nvram)]
                    (host state side (get-in @state [:runner :identity]) c {:facedown true}))))}
 
+   "Reboot"
+   (letfn [(install-cards [state side eid card to-install]
+             (if (empty? to-install)
+               (do
+                 (move state side (find-latest state card) :rfg)
+                 (effect-completed state side eid))
+               (let [f (first to-install)]
```
Comment (Saintis) [inline src/clj/game/cards/events.clj]: You might be able to make this an `if-let` and therefore save a layer of indentation.

File: src/clj/game/cards/events.clj [hunk 2]
```diff
@@ -1544,6 +1544,30 @@
...
+                 (system-msg state :runner (str "uses Reboot to install " (card-str state f) " facedown"))
```
Comment (nealterrell) [inline src/clj/game/cards/events.clj]: Can we condense this into a single list of all 5 chosen cards? 5 log messages seems excessive.

File: src/clj/game/cards/events.clj [hunk 3]
```diff
@@ -1544,6 +1544,30 @@
...
+                 (wait-for (runner-install state :runner f {:facedown true :no-msg true})
+                           (install-cards state side eid card (rest to-install))))))]
+     {:req (req archives-runnable)
+      :effect (effect
+                (run :archives
+                     {:req (req (= target :archives))
+                      :replace-access
+                      {:prompt "Choose up to five cards to install"
```
Comment (nealterrell) [inline src/clj/game/cards/events.clj]: This effect should be `:mandatory true`.

**Pull request merged**